tests/Aql/WaitingExecutionBlockMock.h

```diff
@@ -45,16 +45,27 @@ namespace aql {
  */
 class WaitingExecutionBlockMock final : public arangodb::aql::ExecutionBlock {
  public:
+  /**
+   * @brief Define how often this Block should return "WAITING"
+   */
+  enum WaitingBehaviour {
+    NEVER,  // Never return WAITING
+    ONCE,  // Return WAITING on the first execute call, afterwards return all blocks
+    ALLWAYS  // Return Waiting once for every execute Call.
```

Comment:
```suggestion
    ALWAYS  // Return Waiting once for every execute Call.
```

Reply:
This has to be applied throughout of course

Reply:
I guess I will never learn to type this correct...